react/src/components/FolderCreateModal.tsx

```diff
@@ -135,6 +135,9 @@ const FolderCreateModal: React.FC<FolderCreateModalProps> = ({
             document.dispatchEvent(
               new CustomEvent('backend-ai-folder-list-changed'),
             );
+            document.dispatchEvent(
+              new CustomEvent('backend-ai-folder-updated'),
+            );
```

Comment:
What is the difference between 'backend-ai-folder-list-changed' and 'backend-ai-folder-updated'? If we need to add `backend-ai-folder-updated`, how about changing it to something more specific, such as `backend-ai-folder-created`?